app/controllers/hunters_controller.rb

```diff
@@ -18,8 +18,10 @@ def index
   # @param :available_improvements [Boolean] Toggles whether or not to include a list of improvements available to
   #   the hunter
   # @see Hunter#available_improvements
+  # Available_gear contains all gear available to the Hunter's current playbook
   def show
     @available_improvements = params[:available_improvements]
+    @available_gear = @hunter.playbook.gears
```

Comment:
Where do we use `@available_gear`?

Reply:
We do not.  This was added to address the last bullet point on the issue, which ended up not being needed.  Removed in commit c6538bd.